_resources/participatory-design-inclusivity-heuristics.md

```diff
@@ -0,0 +1,248 @@
+---
+layout: heuristics
+title: Participatory design&#58; inclusivity heuristics
+permalink: /resources/participatory-design-inclusivity-heuristics/
+redirect_from:
+  - /participatory-design-inclusivity-heuristics/
+description: A set of heuristics for ensuring that a participatory design effort is proceeding in an inclusive way.
+---
+
+<h1>Participatory Design: Inclusivity Heuristics</h1>
+
+<p>The following indicators can be used to help determine if a <a href="{{site.baseurl}}/fundamentals/participatory-design">participatory design</a> process is proceeding in a way that encourages input from stakeholders and users. Indicators are spread across three broad categories: <a href="#account-for-context">Account for context</a>, <a href="#encourage-participation">Encourage participation</a>, and <a href="#inform-and-share-understanding">Inform and share understanding</a>.</p>
+
+<table>
+  <thead>
+    <tr>
+      <th scope="col" class="col-indicator">Indicator</th>
+      <th scope="col" class="col-why-it-matters">Why it matters</th>
+      <th scope="col" class="col-good-signs">Good signs</th>
+      <th scope="col" class="col-warning-signs">Warning signs</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th colspan="4" id="account-for-context"><p>Account for context</p></th>
+    </tr>
+    <tr>
+      <th scope="row" class="col-indicator"><p>Values</p></th>
+      <td class="col-why-it-matters"><p>Values tie people together in ways that are difficult to break, especially when they’re not made explicit. Accounting for values helps teams approach design honestly and respectfully.</p></td>
+      <td class="col-good-signs"><p>The team identifies and reflects on its values. The team respects the values of the people affected by its work. The team acknowledges when it seeks to challenge the values of others.</p></td>
```

Comment:
How would you identify values? What does it mean to reflect on them? How do you know you're respecting other people's values or challenging them?

Reply:
I appreciate the question. I'm going to sketch a few different thoughts which I can hopefully incorporate back into the document itself.

- 18F kickoff meetings usually start with some variation of Hopes + Fears, "What does success look like?" or a "pre-mortem" exercise that describes a failure state to avoid. These aren't necessarily value systems, but they do represent directions the project should go.
- Code for America's fellowship saw teams of three working together for nearly a year on a project. A lot can go wrong over a year, so each fellowship team was asked to start by creating a [team constitution](https://peopledevelopmentmagazine.com/2015/11/16/create-a-constitution/) outlining its values, norms, etc.
- Design principles also come to mind here. 18F has [a guide on this](https://design-principles-guide.18f.gov/index.html), though I haven't used it on a project. GDS has some [great design principles](https://www.gov.uk/guidance/government-design-principles) that animate their work.
- Here's [one exercise](http://www.liberatingstructures.com/33-purpose-to-practice-p2p/) from liberating structures that seems relevant.

What comes to mind for you, Liz? 

In terms of reflecting, I think reflecting on values means actually assessing/re-assessing how well a team is following its own values by doing something like Dave Gray's [culture mapping](http://www.xplaner.com/culturemap/) exercise.

Finally, in terms of respecting/challenging others' values, I think you have to do research and spend time listening to customers to understand their values and the degree to which your team's efforts are being well received.